src/assets/wise5/components/aiChat/ai-chat-student/ai-chat-student.component.ts

```diff
@@ -51,12 +58,96 @@ export class AiChatStudentComponent extends ComponentStudent {
 
   ngOnInit(): void {
     super.ngOnInit();
+    if (this.component.isComputerAvatarEnabled()) {
+      this.initializeComputerAvatar();
+    } else {
+      this.computerAvatar = this.computerAvatarService.getDefaultAvatar();
+    }
     if (this.componentState != null) {
       this.messages = this.componentState.studentData.messages;
     }
     this.initializeMessages();
   }
 
+  initializeComputerAvatar(): void {
```

Comment:
Add access modifier?
```suggestion
  private initializeComputerAvatar(): void {
```